src/libraries/System.Private.CoreLib/src/System/Collections/Hashtable.cs

```diff
@@ -258,7 +262,7 @@ public Hashtable(int capacity) : this(capacity, 1.0f)
         public Hashtable(int capacity, float loadFactor)
         {
             if (capacity < 0)
-                throw new ArgumentOutOfRangeException(nameof(capacity), SR.ArgumentOutOfRange_NeedNonNegNum);
+                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.capacity, ExceptionResource.ArgumentOutOfRange_NeedNonNegNum);
```

Comment:
What's wrong with keeping `nameof` as the argument?

Reply:
2566346[[_**234567**_]()]()